pinot-common/src/main/java/org/apache/pinot/common/http/MultiHttpRequestResponse.java

```diff
@@ -18,38 +18,26 @@
  */
 package org.apache.pinot.common.http;
 
+import java.io.IOException;
 import java.net.URI;
 import org.apache.http.client.methods.CloseableHttpResponse;
-
-
 public class MultiHttpRequestResponse implements AutoCloseable {
```

Comment:
(minor) Can we add some empty lines between methods? Auto-reformat should do the job